### DISCUSSION TITLE
No Decode Delegate for OpenRaster (ORA)

### DISCUSSION THREAD

**Discussion by SethFalco** [ImageMagick/ImageMagick#3184] [Help]
I can see that recently ImageMagick got support for OpenRaster via the commit history and website.

![image](https://user-images.githubusercontent.com/22801583/106616021-5bbebe80-656d-11eb-8256-de3ea5b3f06c.png)

I'm currently trying to export an ORA file as a PNG using ImageMagick, however whenever I attempt to do so:

```bash
seth@Rogeliina:~/Downloads$ magick pandaAww.ora out.png
magick: no decode delegate for this image format `ORA' @ error/constitute.c/ReadImage/572.
```

Sorry, it's unclear to me what I need to install or do to enable this functionally.
How can I get ImageMagick (and/or Magick.NET) to read OpenRaster files?

**Comment by dlemstra** [comment]
To make it possible to read this format in the Magick.NET library or with ImageMagick you need the `libzip` delegate. This should be added to the Magick.Native library. Can you open an issue here: https://github.com/dlemstra/Magick.Native to request this?
